source/deployment/mobile-app-deployment.rst

```diff
@@ -0,0 +1,571 @@
+
+    **Note:** `Only Mattermost Enterprise Edition E10 <https://about.mattermost.com/pricing/>`_ or higher to receive access to our Hosted Push Notification Service (HPNS)
+
+If you will be using the Mattermost applications via the iOS and Android app stores, the HPNS is all you need.
+
+Our Hosted Push Notification Service offers:
+
+- Access to a publicly-hosted Mattermost Push Notification Service (MPNS).
+- An explicit `privacy policy <https://about.mattermost.com/hpns-privacy/>`_ for the contents of unencrypted messages.
+- Encrypted TLS connections:
+    - between HPNS and Apple Push Notification Services
+    - HPNS and Google’s Firebase Cloud Messaging service
+    - HPNS and your Mattermost server
+- Production-level uptime expectations
+- Compatibility with EMM Providers*
+
+\* *When using our publicly available app store applications and the AppConfig standard.*
+
+Of course, if you would like to host your own version of the MPNS, that is always an option. The `Choosing the Right Deployment Model <https://docs.google.com/document/d/1BhHla4medUGVSWuBPNuoz_h5f85MdlIQWcRn5N2WUos/edit#heading=h.6zz62c4mus3l>`_ section of this guide, as well as our `Mobile App Admin Documentation <https://docs.mattermost.com/mobile/mobile-hpns.html>`_ are the best places to start.
+
+********************************************
+Enterprise Mobile Management (EMM) Providers
+********************************************
+
+EMM Providers develop software that helps enterprise teams manage secure mobile technology deployments. This includes the use of mobile devices and usage ready applications.
+
+Most large enterprise teams are familiar with `Enterprise Mobile Management <https://en.wikipedia.org/wiki/Enterprise_mobility_management>`_ providers, or EMMs. If this is a new term for you, review `Appendix B <https://docs.google.com/document/d/1BhHla4medUGVSWuBPNuoz_h5f85MdlIQWcRn5N2WUos/edit#heading=h.pn9oiqr3rebj>`_ for a list of providers and relevant information.
+
+For those taking on larger deployments, we assume you are already using an EMM Provider.
+
+The AppConfig standard is the critical element when deploying Mattermost via an EMM.
+
+AppConfig is newer, more modern approach when compared to App Wrapping.
+
+We’ll say it several times throughout this guide, and in the next section. If you are using an EMM provider, the only supported path is via AppConfig.
+
+While “App Wrapping” has been widely supported, **it is not recommended**. Further, **it is not a supported path** for Mattermost mobile deployment.
+
+AppConfig (Supported) vs. App Wrapping (Not Supported)
+======================================================
+
+Mattermost only supports the AppConfig standard. It does not support app wrapping. If you proceed with app wrapping, it will lead to an inability to use the mobile app.
```

Comment:
I wouldn't say  `inability to use the mobile app.` but something along the lines of `at your own risk of success`

Reply:
For this audience and the goal of this doc (this is not intended to exist within the documentation for long), we took an approach to be use language to discourage this path. I stick by that, unless there are strong opinions against it.

Reply:
First, and very important, there is no lie here, attempted or implicit. I don't think think it's reasonable to suggest that anyone has made even an attempt. This particular section was written and then rewritten to reflect what was being seen in the field. Original wording used "may" and feedback was that this didn't clearly state the risk involved. In these scenarios customers/users can waste many hours going down this path, only to fail. There are countless examples, including the section preceding this one where we explain why. We can make a change back, but should note that is in conflict with what is being encountered in the field, and a revert from previously approved wording.

Reply:
resolved in meeting